README.md

```diff
@@ -277,18 +277,25 @@ Compose.prototype.map = function(f) {
 };
 ```
 
-#### `sequence` method
+#### `traverse` method
 
 ```hs
-sequence :: Apply f, Traversable t => t (f a) ~> (b -> f b) -> f (t a)
+traverse :: Apply f, Traversable t => t a ~> ((a -> f b), (c -> f c)) -> f (t b)
```

Comment:
Should this be `Applicative f` rather than `Apply f`? `c -> f c` is `of` essentially, isn't it?